oml/metrics/embeddings.py

```diff
@@ -170,17 +172,16 @@ def _calc_matrices(self) -> None:
         if self.postprocessor is None:
             pass
         elif isinstance(self.postprocessor, PairwiseEmbeddingsPostprocessor):
-            max_k = max([*self.cmc_top_k, *self.precision_top_k, *self.map_top_k])
-            if max_k > self.postprocessor.top_n:
-                warn(
-                    f"One of retrieval metrics will be computed at k = {max_k},"
-                    f"but postprocessor will re-rank only {self.postprocessor.top_n} closest galleries."
-                    f"Make sure that this is the desired behaviour."
-                )
```

Comment:
Sorry, I can't find where you decided to put this warning. Or did you get rid of them at all?